nixos/modules/services/mail/postfix.nix

```diff
@@ -25,6 +25,8 @@ let
 
   clientRestrictions = concatStringsSep ", " (clientAccess ++ dnsBl);
 
+  smtpTlsSecurityLevel = if cfg.useDane then "dane" else "may";
```

Comment:
I don't have a better name, but this value is used for `smtp_tls_security_level` and `smtpd_tls_security_level`. Maybe `computedTlsSecurityLevel` or only `tlsSecurityLevel`?

Reply:
I would go for `tlsSecurityLevel`.

Reply:
Oh geeze -- I totally missed that little `d` hiding in there!

I checked the manual, and it turns out that `"dane"` is not a valid value for the `smtpd_...` option, only the `smtp_...` option.  So, I've just left the former to always be `"may"`.